src/style.css

```
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-bar__input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-weight: 400;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 1px 8px rgba(74, 114, 160, 0.2);
+}
+
+.bigInput {
+  font-size: 16px;
+  padding-left: 62px;
+  height: 70px;
+  background-size: 19px;
+  background-position: 25px 26px;
+}
+
+.smallInput {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 12px 14px;
+}
+
+.search-bar__input:focus {
+  border-radius: 3px;
+  font-weight: 900;
+  background-color: transparent;
+}
+
+.search-bar__input::placeholder {
+  font-weight: 400;
```

Please pay attention to my comments
```suggestion
  font-weight: 300;
```